projects/addon-doc/components/documentation/documentation.template.html

```diff
@@ -168,22 +143,22 @@
                                 (ngModelChange)="propertyConnector.onValueChange($event || 0)"
                             />
 
-                            <ng-container *ngSwitchCase="'PolymorpheusContent'">
-                                <tui-primitive-textfield
-                                    *ngIf="
-                                        propertyConnector.documentationPropertyValue
-                                            | tuiIsPrimitivePolymorpheusContentPipe
-                                    "
-                                    tuiTextfieldSize="m"
-                                    [nativeId]="propertyConnector.attrName"
-                                    [tuiTextfieldCleaner]="true"
-                                    [tuiTextfieldLabelOutside]="true"
-                                    [value]="propertyConnector.documentationPropertyValue.toString()"
-                                    (valueChange)="propertyConnector.onValueChange($event)"
-                                >
-                                    Default content
-                                </tui-primitive-textfield>
-                            </ng-container>
+                            <!--                            <ng-container *ngSwitchCase="'PolymorpheusContent'">-->
```

Comment:
should it be removed or uncommented?

Reply:
I don't know yet, probably removed. I didn't find any usage due to the nested ngIf never being true. I didn't look to closely though, but anyway, we can probably drop it and later improve docs at the spot if we find a need to do so. I'll drop it in my next PR.